lib/Dialect/HW/HWOps.cpp

```diff
@@ -754,16 +754,12 @@ Operation *InstanceOp::getReferencedModule(const SymbolCache *cache) {
       return result;
 
   auto topLevelModuleOp = (*this)->getParentOfType<ModuleOp>();
-  if (!topLevelModuleOp)
-    return nullptr;
 
   return topLevelModuleOp.lookupSymbol(moduleName());
 }
 
 // Helper function to verify instance op types
 static LogicalResult verifyInstanceOpTypes(InstanceOp op, Operation *module) {
-  assert(module && "referenced module must not be null");
```

Comment:
Why remove this?

Reply:
It seemed redundant with:

https://github.com/llvm/circt/blob/537d067e7e47e0911a90b6354ae81ef04e8e78af/lib/Dialect/HW/HWOps.cpp#L853-L855

Reply:
ok